Formula/haxe.rb

```diff
@@ -32,7 +32,8 @@ def install
       ENV["OPAMROOT"] = buildpath/"opamroot"
       ENV["OPAMYES"] = "1"
       system "opam", "init", "--no-setup"
-      system "opam", "install", "ocamlfind"
+      system "opam", "switch", "4.04.2"
```

Comment:
This is the PR waiting to be merged in upstream prmap: https://github.com/backtracking/ptmap/pull/4
I don't know much about how homebrew or opam works, is it possible to add a patch to a formula? Would that even make sense?